theme/fonts.js

```diff
@@ -1,19 +1,19 @@
-const baseFontSize = 16
-const baseLineHeight = 20
+const baseFontSize = 18
+const baseLineHeight = 22
```

Comment:
this has helped on the emulators however the problem of how each OS handles font sizes remains

Reply:
can we utilise `Constants.expoConfig` somehow to find out if it is android or iOS and set these values accordingly? would that reduce the difference between both OS?

Reply:
That's a good plan although I already read a little on the  but I found when testing with both emulated devices (something I wasn't able to do before) it is a problem for both OS's so thought would up the base anyway

Reply:
ah i see, so it is not only a problem of OS differences...